chi2/chinotype.py

```diff
@@ -693,7 +771,7 @@ def chi2_output(self, db):
         select prefix, ccd, name, {3}, frc_{3}, {0}, frc_{0}, chisq, dir
         from ranked_data {2}
         '''.format(self.chi_name, self.pcounts, limstr, self.ref, filterStr, cutoff)
-        cols, rows = do_log_sql(db, sql, self.filter)
+        cols, rows = do_log_sql(db, sql)
```

Comment:
`do_log_sql(db, sql, self.filter)` produces an error. If the code ran without error before it's because this functionality was not being exercised. Checked the declaration of `do_log_sql` and saw nothing that would even use a filter arg, the third optional argument seems to be about deciding between `execute()` and `executemany()`. Removed third argument and errors stopped.